components/x-teaser-timeline/bower.json

```diff
@@ -3,8 +3,8 @@
   "private": true,
   "main": "dist/TeaserTimeline.es5.js",
   "dependencies": {
-    "o-colors": "^4.7.2",
-    "o-grid": "4.4.4",
-    "o-typography": "^5.7.5"
+    "o-typography": "^6.4.5",
+    "o-colors": "^5.3.0",
+    "o-grid": "^5.2.9"
```

Comment:
is this `bower.json` only used for local development? i've noticed that the apps using `x-teaser-timeline` (`next-myft-page` and `ft-app`) both have different versions of these components, so this probably isn't actually a breaking change?

Reply:
oh, i see, they're being used in the component's Sass files which are compiled to CSS before reaching the apps